ocaml/xapi/storage_migrate.ml

```diff
@@ -339,7 +318,10 @@ let vdi_info x =
       failwith "Runtime type error: expecting Vdi_info"
 
 module Local = StorageAPI (Idl.Exn.GenClient (struct
-  let rpc call = rpc ~srcstr:"smapiv2" ~dststr:"smapiv2" (local_url ()) call
+  let rpc call =
+    Storage_utils.rpc ~srcstr:"smapiv2" ~dststr:"smapiv2"
```

Comment:
Should we have a module-local binding for the `smapiv2` string and use that everywhere? That would eliminate any possibility to spell it wrong.

Reply:
This is only for logging, and not used very consistently. I think we should sort this out properly when we get down to do proper distributed tracing. This stuff may then become obsolete.